assets/src/design-system/components/snackbar/stories/index.js

```diff
@@ -146,3 +147,32 @@ export const LongMessage = () => (
     ]}
   />
 );
+
+export const ThumbnailMessage = () => {
+  const success = boolean('success');
+  const landscape = boolean('landscape');
+
+  return (
+    <SnackbarContainer
```

Comment:
nit: presenting this with the theme used by the editor (dark instead of light) might be good, you can wrap the snackbar container with this storybook util
```
<DarkThemeProvider>
```

Still looks great! 
<img width="721" alt="Screen Shot 2021-06-15 at 10 11 58 AM" src="https://user-images.githubusercontent.com/10720454/122095379-2a2eed00-cdc2-11eb-8856-ef6e16f12a8a.png">